src/ui/server/controllers/insurance/your-buyer/your-buyer-details/validation/yourBuyerDetailsValidation.ts

```diff
@@ -0,0 +1,13 @@
+import generateValidationErrors from '../../../../../helpers/validation';
+import { RequestBody } from '../../../../../../types';
+import { yourBuyerPageVariables } from '../../../../../constants/field-ids/insurance/your-buyer';
+
+export const yourBuyerDetailsValidation = (responseBody: RequestBody) => {
+  const { countryName } = responseBody;
+  const YOUR_BUYER_PAGE_VARIABLE = yourBuyerPageVariables();
+  let updatedErrors;
+  if (!countryName) {
+    updatedErrors = generateValidationErrors(YOUR_BUYER_PAGE_VARIABLE.FIELDS.BUYER_COUNTRY.ID, YOUR_BUYER_PAGE_VARIABLE.FIELDS.BUYER_COUNTRY.ERROR_MESSAGE);
+  }
+  return updatedErrors;
+};
```

Comment:
Can we use the same pattern/approach as other validation please e.g https://github.com/UK-Export-Finance/exip/tree/main-application/src/ui/server/controllers/insurance/policy-and-export/single-contract-policy/validation

Need a rules directory and just importing field IDs and error messages from constants